.github/workflows/pr-auto-labeler.yml

```
@@ -1,15 +1,15 @@
-
-jobs:
-  labeler:
-    permissions:
-      contents: read
-      pull-requests: write
-    runs-on: ubuntu-22.04
-    steps:
-    - uses: actions/labeler@v5
-      with:  
-        repo-token: ${{ secrets.GITHUB_TOKEN }}  
+name: "Pull Request Labeler"
```

yamllint

[error] wrong new line character: expected \n

(new-lines)